src/Tribe/Views/V2/View.php

```diff
@@ -755,25 +755,37 @@ public function next_url( $canonical = false, array $passthru_vars = [] ) {
 
 		$url = $this->get_url();
 
+		$query_args = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 134e75a5772df713a13580b1deca1580</sub>